lib/sql.js

```diff
@@ -229,7 +230,7 @@ SQLConnector.prototype.tableEscaped = function(model) {
  * @returns {String} The escaped column name
  */
 SQLConnector.prototype.columnEscaped = function(model, property) {
-  return this.escapeName(this.column(model, property));
+  return this.tableEscaped(model) + '.' + this.escapeName(this.column(model, property));
 };
```

Comment:
@kolach Good catch. We need to fix it. Maybe we should introduce something like:
```
SQLConnector.prototype.qualifiedColumnEscaped = function(model, property) {
}
```